tests/Util/Age/AgeUnderOverProcessorTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace YotiTest\Util\Age;
+
+use YotiTest\TestCase;
+use Yoti\Util\Age\AgeUnderOverProcessor;
+
+class AgeUnderOverProcessorTest extends TestCase
+{
+    public $processor;
+    public $dummyProfile;
+
+    public function setUp()
+    {
+        $this->dummyProfile = [
+            'family_name' => 'TestFamilyName',
+            'given_names' => 'TestGivenName',
+            'full_name'   => 'TestGivenName TestFamilyName',
+            'date_of_birth' => '11-07-2017',
```

Comment:
I know it doesn't matter, but can you add a date so the fake profile is over 18?